packages/web/hooks/user-settings/use-user-favorite-asset-denoms.ts

```diff
@@ -0,0 +1,33 @@
+import { useCallback } from "react";
+
+import { useLocalStorageState } from "../window/use-localstorage-state";
+
+export function useUserFavoriteAssetDenoms() {
+  const [favoritesList, onSetFavoritesList] = useLocalStorageState(
```

Comment:
Since `onSetFavoritesList` is not an event, but a set function I believe we should remove the `on`
```suggestion
  const [favoritesList, setFavoritesList] = useLocalStorageState(
```